src/2ls/preprocessing_util.cpp

```diff
@@ -22,6 +22,8 @@ Author: Peter Schrammel
 
 #include "2ls_parse_options.h"
 
+#define NOT_MATH_FUN(call, fun) call != fun &&call != fun "f" && call != fun "l"
```

Comment:
Missing space, is that enforced by clang-format?
```suggestion
#define NOT_MATH_FUN(call, fun) call != fun && call != fun "f" && call != fun "l"
```